### ISSUE TITLE
Order of products not correct on front end when using "position inside category" setting 

### ISSUE THREAD

**Issue by gingerling** [thirtybees/thirtybees#656]
I have my site set to display products on the front end by the order within the category on the back end. 

I have changed the order on products for the category vegan (https://www.ethicalpets.co.uk/vegan) significantly, but the change is not applying on the front end.

I have changed the position inside category to other settings, and they all seem to apply fine - even position inside category - descending works fine, however, position in category ascending simply does not apply. I have cleared cache.

**Comment by yaniv14** [comment]
This is a known bug if you are using Block layered navigation module.
The module override the function that order the products.
If you are using this module, try disabling it just to confirm what I am saying.

**Comment by gingerling** [comment]
Yes this is correct! Thank you. I don't actually know what this module does, is it needed? Should I just disable it or is there another way to fix it, or does it need a patch?

If you can give me a little more info, I will do some documentation.

**Comment by yaniv14** [comment]
This module is the one that adds the option to filter products in a category based on price range, features, attributes, etc....

**Comment by gingerling** [comment]
Yes this is correct! Thank you. I don't actually know what this module does, is it needed? Should I just disable it or is there another way to fix it, or does it need a patch?

If you can give me a little more info, I will do some documentation.

**Comment by gingerling** [comment]
Ok, so is this a bug report to fix it the problem is there another one somewhere else? Is there any way to fix my issue at all? Thanks

**Comment by gingerling** [comment]
Hi, In the end I turned off that block - it's a rock/hardplace issue really, it's quite essential, but especially when products have low stock, it's so essential to control the order they display in. How fixable is this? Is it something a small bounty would help with?

**Comment by gingerling** [comment]
Hi, this bug now seems to be in operation without the module.

The module is disabled:

![Screenshot from 2019-12-19 11-51-16](https://user-images.githubusercontent.com/2853040/71171597-14387e80-2256-11ea-86f4-cf4500ed8067.png)

Here is a category

![Screenshot from 2019-12-19 11-51-23](https://user-images.githubusercontent.com/2853040/71171615-1dc1e680-2256-11ea-97c3-b9eeb068e2df.png)

Here are is the desired order:

![Screenshot from 2019-12-19 11-51-33](https://user-images.githubusercontent.com/2853040/71171630-2aded580-2256-11ea-9357-ca28036d3aaf.png)

Here is the actual order:

![Screenshot from 2019-12-19 11-51-49](https://user-images.githubusercontent.com/2853040/71171649-316d4d00-2256-11ea-83ac-7cde07ae8c96.png)

The second product is out of stock and I urgently need to move it away from that position, but it won't shift. I tried clearing the cache.

I'm on version 1.1.0

**Comment by gingerling** [comment]
I hid the above comment, made an error - did not notice products were sorted by name
